web/components/logos/NavBarLogo.tsx

```diff
@@ -0,0 +1,50 @@
+import Link from 'next/link';
+import Image from 'next/image';
+import navbar_logo from "../../public/navbar_logo.png";
+import navbar_logo_bg from "../../public/navbar_logo_bg.png"
+
+interface LogoProps {
+  isAdmin: boolean;
+  caregiverName?: string;
+}
+
+const Logo: React.FC<LogoProps> = ({ isAdmin, caregiverName }) => {
+  const homeLink = isAdmin ? '/admin' : '/caregiver';
+  
+  return (
+    <div>
+        <Link href={homeLink}>
+        <button>
+            <span className="left-0 right-0 w-[318px] h-[81px]">
```

Comment:
The navbar should take up a percentage of the screen instead of a fixed width since people have different screen sizes on laptops, monitors, and tablets